drivers/storage/portworx/components_test.go

```diff
@@ -2791,6 +2797,10 @@ func TestSecurityInstall(t *testing.T) {
 		Name:  pxutil.EnvKeyPortworxAuthJwtSharedSecret,
 		Value: string(sharedSecretKey),
 	})
+	cluster.Spec.Env = append(cluster.Spec.Env, v1.EnvVar{
```

Comment:
nit: Can we duplicate this case and also check that if user has given only one env (either system-secret or apps-secret), then we don't overwrite the other? I see from the code we do handle that, but just to make sure we don't regress in future and overwrite one because of the other.